tests/diag.sh

```diff
@@ -110,7 +110,7 @@ function generate_conf() {
 	fi
 	echo imdiag running on port $new_port
 	echo "module(load=\"../plugins/imdiag/.libs/imdiag\")
-global(inputs.timeout.shutdown=\"10000\")
+global(inputs.timeout.shutdown=\"60000\")
```

Comment:
Is this actually necessary? Looks overly excessive. Please provide a reason.

Reply:
I had the impression that the input timeout aborts full processing in some cases. Can be set back to 10000 if tests run stable with it.

Reply:
ok, let's keep it for now. It should actually not have bad side-effects.